### PR TITLE
Run PBnJ v1 HTTP alongside gRPC

### PR DESCRIPTION
**Pull request by jacobweinstock**
## Description

This adds the PBnJv1 HTTP server to run alongside the
gRPC server.

## Why is this needed

backward compatibility

Fixes: #

## How Has This Been Tested?
manual power status test was successful.

## How are existing users impacted? What migration steps/scripts do we need?

n/a

## Checklist:

I have:

- [X] updated the documentation and/or roadmap (if required)
- [X] added unit or e2e tests
- [X] provided instructions on how to upgrade

### CODE REVIEW

File: scripts/run-image.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+# trap ctrl-c and call ctrl_c()
+trap ctrl_c INT
+
+function ctrl_c() {
+    docker rm -f ${CONTAINER_ID}
+    exit 0
+}
+
+CONTAINER_ID=$(docker run -d -e ACCESS_ID=1234 -e ACCESS_SECRET=1234 -e PNBJ_LOGLEVEL=debug -p 9090:9090 -p 50051:50051 pbnj:local)
```
Comment (mikemrm) [inline scripts/run-image.sh:15]: Maybe we should expose :8080 which is the metrics and http healthcheck endpoint as well?

**Review by mikemrm — APPROVED** [review]

**Comment by gianarb** [comment]
What's the use case? I am asking because tink-server has the same feature, but it undocumented and I personally don't see these multiple ways of interacting with tink a plus, even more, if there is no documentation, clients, and "care".

Can we have only one? I am ok with what ever

**Comment by jacobweinstock** [comment]
> What's the use case? I am asking because tink-server has the same feature, but it undocumented and I personally don't see these multiple ways of interacting with tink a plus, even more, if there is no documentation, clients, and "care".
> 
> Can we have only one? I am ok with what ever

Fair questions. This PR is definitely lacking in docs/comments around context. Apologies for that. I will add documentation to the PR for this info. This PR has a few ideas/goals.
- allow PBnJ.next to be backward compatible
- ease the upgrade path
- provide the additional functionality of user management to v1 users
- be a temporary step toward deprecating v1, this dual-stack approach will have a documented end of life

I definitely agree there are trade-offs with this approach. Just brainstorming here, what if we add a flag for running v1 and have it off by default?

File: scripts/run-image.sh (hunk 1 above)
Comment (jacobweinstock) [reply scripts/run-image.sh:15]: yeah, good point. will do

**Comment by gianarb** [comment]
I am less confused but apparently, PBNJ is just getting inline with the rest of the project. A bit of new code and ideas mixed with old things for compatibility with Packet.

I am not really sure this should be a "tinkerbell oss project business" (the kind of business we should care about in github.com/tinkerbell) but ok 👍

File: scripts/run-image.sh (hunk 1 above)
Comment (jacobweinstock) [reply scripts/run-image.sh]: updated

**Review by mikemrm — APPROVED** [review]

**Pull request merged**